### PR TITLE
fixed emoji issue, now emojis are working

### PR DESCRIPTION
**Pull request by LalitNM**
@abhigyank have a look.

### CODE REVIEW

**Comment by abhigyank** [comment]
@LalitNM  Good work! This is how we'd want this to be done. Spot on! 
I am wondering how the emobox gets the icons, any idea? I think it maybe good to have emojis in the normal order that is usually used, what do you think?

There is one bug you need to take care of, it is that what if the work between `::` is not a emoji, like if you message `:thisisnotanemoji:` , in that case it should just show `:thisisnotanemoji:` in the sent message. Can you take care of this?

**Comment by LalitNM** [comment]
There is a list in `emojify.js`, which has all emoji names as strings, emoji name is name of png image without `.png`. And a for loop is used to add all emojis in `index.ejs` file. This is how emobox gets icons.

For ordering of emojis I am thinking that I should add different folders for different categories of emojis, but for that I need to know that from where you downloaded these png files(I searched but could't get a satisfactory result, and I don't want to handpick emojis :sweat_smile:). After categorising emojis I will style emobox similar to what zulipchat's emoji-box is.

And for :thisisnotanemoji: issue I will try to fix that

File: public/client.js [hunk 1]
```diff
@@ -98,18 +98,39 @@ socket.on('Display Message', function(data) {
...
+    while (colon1 != -1) {
+      var colon2 = p.indexOf(":", colon1 + 1);
+      if (colon2 != -1) {
+        emoji = p.slice(colon1+1, colon2);
+        if (emojiList.includes(emoji)){
+          var p = p.slice(0, colon1) + "<img class=\"emoji\" src=\"images/emoji/" + emoji + ".png\">" + p.slice(colon2 + 1);
+          colon1 = p.indexOf(":", colon2 + 1);
+        }
+        else {
+          break
```
Comment (abhigyank) [inline public/client.js]: I don't think this should be a break here.
Otherwise `:notanemoji: followed by :emoji:` will not work

Comment (LalitNM) [reply public/client.js]: I will again look into that and will find a better solution.

Comment (LalitNM) [reply public/client.js]: I fixed this in 1deea3d.

**Comment by abhigyank** [comment]
Good work @LalitNM on this! :rocket: 
Closes #28

**Pull request merged**